reconcile/openshift_resources.py

```diff
@@ -274,8 +329,11 @@ def run(dry_run=False):
             logging.info(['apply', cluster, namespace,
                           resource_type, d_item.name])
 
-            logging.debug("CURRENT: " +
-                          OR.serialize(OR.canonicalize(c_item.body)))
+            if c_item is None:
+                logging.debug("CURRENT: None")
+            else:
+                logging.debug("CURRENT: " +
+                              OR.serialize(OR.canonicalize(c_item.body)))
```

Comment:
thanks for catching this bug!

Reply:
can you submit this as a separate PR, so I can merge, and you can rebase?
I need the fix for this ASAP in the master branch, as running integrations would fail now.

Reply:
Done!

Reply:
added as part of #20